### DISCUSSION TITLE
blog/2022/220407-certbot/

### DISCUSSION THREAD

**Discussion by giscus** [zhanbao2000/blog#1] [Announcements]
# blog/2022/220407-certbot/

Arisa 的个人博客

https://blog.arisa.moe/blog/2022/220407-certbot/

**Comment by MBRjun** [comment]
~~人生苦短，我用 [acme.sh](https://acme.sh)~~

**Reply by zhanbao2000** [reply]
我现在终于改用acme.sh了（
